3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestIntegrationTest.scala

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright (c) 2012-2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.common
+package enrichments
+package registry
+package apirequest
+
+// json4s
+import org.json4s.JsonAST._
+import org.json4s.jackson.{ parseJson, prettyJson }
+
+// specs2
+import org.specs2.Specification
+import org.specs2.scalaz.ValidationMatchers
+import org.specs2.matcher.Matcher
+
+// Iglu
+import com.snowplowanalytics.iglu.client.SchemaKey
+
+// This project
+import outputs.EnrichedEvent
+import utils.JsonUtils
+
+object ApiRequestEnrichmentIntegrationTest {
+  def continuousIntegration: Boolean = sys.env.get("CI") match {
+    case Some("true") => true
+    case _ => false
+  }
```

Comment:
The integration tests will only run if the CI environment variable is set to "true". Do you think we should do this differently, for example by having an sbt command to run integration tests? This would be faster than manually setting CI=true and starting the Python server process whenever you want to run integration tests locally.

What you have at the moment works fine, so maybe my suggestion should be a ticket for a future release.